src/call.c

```diff
@@ -790,8 +793,18 @@ int call_alloc(struct call **callp, const struct config *cfg, struct list *lst,
 	call->af     = prm->af;
 	call->ansadir = SDP_SENDRECV;
 	call->ansvdir = SDP_SENDRECV;
+	call->support_replaces = false;
 	call_decode_sip_autoanswer(call, msg);
 
+	/* check if replaces is supported on incoming calls */
+	hdr = sip_msg_hdr(msg, SIP_HDR_SUPPORTED);
+	if (hdr) {
+		if (!re_regex(hdr->val.p, hdr->val.l, "replaces", &v)) {
+			info("call %s supports attended transfer\n", call->id);
```

Comment:
- Should be a `debug()` line, or remove it after your tests!
- Start the string with `call: `!